src/cookbook/networking/fetch-data.md

```diff
@@ -231,8 +231,10 @@ it's not recommended to put an API call in a `build()` method.
 Flutter calls the `build()` method every time it needs
 to change anything in the view,
 and this happens surprisingly often.
-Leaving the `fetch` call in your `build()` method
-floods the API with unnecessary calls and slows down your app.
+Leaving the `fetch` call in your `build()` method would
```

Comment:
Try reading this line out loud. It feels convoluted to me.

Reply:
You're right, I've updated it!